### PR TITLE
Implement static UI fixes #28

### LINKED ISSUE #28: Create static UI and mock functions

### PR DESCRIPTION
**Pull request by bryancabansay**
Created static UI. Mock functionalities to follow.

### CODE REVIEW

**Review by ctapang — COMMENTED** [review]
Just a single item to note. Looking great!

**Comment by bryancabansay** [comment]
This is now ready for review.

File: src/components/Cards/PriceFeed/index.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return (
+    <Card title="Price" className="cards">
+      <Row>
+        <Col span={2} className="align-to-input">
+          SOL
+        </Col>
+        <Col span={8}>
+          <Input
+            suffix={
+              <Tooltip title={"Current SOL value to USD is " + SOL_VALUE + "."}>
```
Comment (ctapang) [inline src/components/Cards/PriceFeed/index.tsx:17]: Please connect to the completed SOL price oracle in $ZeroInterestDeFi/anchor/oracles/pyth.

**Review by ctapang — APPROVED** [review]
This is most wonderful!

**Comment by ctapang** [comment]
Merging now, so @y2kappa and I can easily clone it in our workstations.

**Pull request merged**